drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -50,4 +50,13 @@ config NORDIC_QSPI_NOR_XIP
 	  QSPI NOR flash chip is executed until the driver has been setup.
 	  This will also disable power management for the QSPI NOR flash chip.
 
+config NORDIC_QSPI_NOR_TIMEOUT_MS
+	int "Timeout for QSPI operations (ms)"
+	default 500
+	help
+	  The QSPI peripheral operation timeout in milliseconds.
+	  Primarily intended for long running operations such as
+	  flash mass erase. 500 ms is the default value as
+	  set in the nrfx QSPI driver.
```

Comment:
This is not a good explanation, nrfx should not be referenced here (Zephyr users may not care about nrfx at all). Just give a brief explanation of why 500ms is a reasonable value.

Reply:
I have updated this to reflect typical sector erase time specs